WindowManager/app/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="colorPrimary">#6200EE</color>
+    <color name="colorPrimaryDark">#3700B3</color>
+    <color name="colorAccent">#03DAC5</color>
+
+    <color name="colorFeatureFold">#7700FF00</color>
```

Comment:
resources tend to be name with undercases, rather than camel case

Reply:
👍